@here/harp-datasource-protocol/test/StyleSetEvaluatorTest.ts

```diff
@@ -112,24 +116,68 @@ describe("StyleSetEvaluator", function() {
         });
     });
 
+    it("generates technique render order from priorities", async () => {
```

Comment:
this test is moved from TileGeometryCreatorTest where the functionality lived before